Content/0.3.2 - Account Settings/accountSettings.json

```diff
@@ -48,6 +49,7 @@
 	"accountSettings_specificAssetsDeposits_removeAsset": "Remove Asset",
 	"accountSettings_specificAssetsDeposits_removeAssetMessageDeny": "The asset will be removed from the deny list",
 	"accountSettings_specificAssetsDeposits_removeAssetMessageAllow": "The asset will be removed from the allow list",
+	"accountSettings_specificAssetsDeposits_removeAssetMessageDepositors": "The badge will be removed from the list",
```

Comment:
Not a requirement but maybe **removeBadgeMessageDepositors** would suit better ?

Reply:
Sure. I wasn't sure if this string was correct, since it mentioned "badge" but the context seemed unrelated. I will change it.